setup.py

```diff
@@ -42,6 +42,7 @@
     python_requires=">=3.6",
     install_requires=requires,
     extras_require={
+        "email": ["pydantic[email]>=1.2"],
```

Comment:
Unsure if this version constraint is redundant since it's already in `requirements.txt`.

Reply:
It's okay to have a redundant version constraint.